github4s/src/main/scala/github4s/interpreters/IssuesInterpreter.scala

```diff
@@ -183,4 +197,23 @@ class IssuesInterpreter[F[_]](implicit client: HttpClient[F], accessToken: Optio
       headers,
       pagination = pagination
     )
+
+  override def listMilestones(
+      owner: String,
+      repo: String,
+      state: Option[String],
+      sort: Option[String],
+      direction: Option[String],
+      pagination: Option[Pagination],
+      headers: Map[String, String]
+  ): F[GHResponse[List[Milestone]]] =
+    client.get[List[Milestone]](
+      accessToken,
+      s"repos/$owner/$repo/milestones",
+      headers,
+      pagination = pagination,
+      params = state.fold(Map.empty[String, String])(s => Map("state"  -> s)) ++
+        sort.fold(Map.empty[String, String])(s => Map("sort"           -> s)) ++
+        direction.fold(Map.empty[String, String])(d => Map("direction" -> d))
```

Comment:
```scala
List(state.map("state" -> _), sort.map("sort" -> _), direction.map("direction" -> _)).flatten.toMap
```

might be a bit better than concatenating maps? I'm not too sure but I feel like there might be a better way to do this 🤔